packages/trader/src/AppV2/Components/OnboardingGuide/guide-tooltip.tsx

```diff
@@ -1,42 +1,75 @@
 import React from 'react';
 import { Button, CaptionText, IconButton } from '@deriv-com/quill-ui';
-import { LabelPairedXmarkSmBoldIcon } from '@deriv/quill-icons';
+import { LabelPairedChevronsUpXlBoldIcon, LabelPairedXmarkSmBoldIcon } from '@deriv/quill-icons';
 import { Localize } from '@deriv/translations';
 import { TooltipRenderProps } from 'react-joyride';
+import { useSwipeable } from 'react-swipeable';
 
-const GuideTooltip = ({ isLastStep, primaryProps, skipProps, step, tooltipProps }: TooltipRenderProps) => (
-    <div {...tooltipProps} className='guide-tooltip__wrapper'>
-        <div>
-            {step.title && (
-                <div className='guide-tooltip__header'>
-                    <CaptionText bold color='var(--component-textIcon-inverse-prominent)'>
-                        {step.title}
+export interface GuideTooltipProps extends TooltipRenderProps {
```

Comment:
In dtrader we usually use types, not interfaces. For me looks good, but we might get extra comments in PR

Reply:
Up to you